io.c

```diff
@@ -15094,64 +15094,72 @@ set_LAST_READ_LINE(VALUE val, ID _x, VALUE *_y)
  *
  *  == Line \IO
  *
- *  You can read an \IO stream line-by-line using these methods:
+ *  \Class \IO supports line-oriented
+ *  {input}[rdoc-ref:IO@Line+Input] and {output}[rdoc-ref:IO@Line+Output]
  *
- *  - IO#each_line: Reads each remaining line, passing it to the given block.
- *  - IO#gets: Returns the next line.
- *  - IO#readline: Like #gets, but raises an exception at end-of-stream.
- *  - IO#readlines: Returns all remaining lines in an array.
+ *  === Line Input
+ *
+ *  \Class \IO supports line-oriented input for
```

Comment:
At [Auto-Linking](https://github.com/ruby/ruby/blob/master/doc/contributing/documentation_guide.md#auto-linking):
> If not, or if you suppress autolinking, consider forcing monofont.

```suggestion
 *  \Class +IO+ supports line-oriented input for
```

"Class" is "the word in question does not refer to a Ruby entity", so it should be rendered as a normal text.
But "IO" is "the reference is to the current class document", it should be rendered as code, I think.

Reply:
@nobu, point taken.  There are many of these in this class overview.  Most should be changed as you've indicated;  some may be better rendered as `I/O`.  If okay with you, I'd like to handle in the next PR instead of complicating matters here.